src/js/components/tablero/NumberCard.jsx

```diff
@@ -1,26 +1,34 @@
 import React from 'react';
+import { SortableElement, sortableHandle } from 'react-sortable-hoc';
 import PropTypes from 'prop-types';
 import './tablero.scss';
 
 function truncateString(value, length) {
   return (value.length > length) ? `${value.substr(0, length - 1)}...` : value;
 }
 
-const NumberCard = ({
+const DragHandle = sortableHandle(() => (
```

Comment:
You have the same thing defined in GraphCard, why you cant use that one? Perhaps this could be a util component.